### PR TITLE
Limit tokens to established GitHub accounts

### PR DESCRIPTION
**Pull request by stephencelis**

### CODE REVIEW

File: Sources/GitHub/Model.swift [hunk 1]
```diff
@@ -56,6 +59,12 @@ public struct GitHubUser: Codable {
   }
 
   public typealias Id = Tagged<GitHubUser, Int>
+
+  private enum CodingKeys: String, CodingKey {
+    case createdAt = "created_at"
```
Comment (stephencelis) [inline Sources/GitHub/Model.swift:64]: Be nice to figure out if we can finally do the snake case stuff...

File: Sources/Database/Database.swift [hunk 2]
```diff
@@ -498,21 +499,24 @@ private struct _Client {
...
     ON CONFLICT ("github_user_id") DO UPDATE
     SET "github_access_token" = $3, "name" = $4
     """,
       [
         email.rawValue,
         envelope.gitHubUser.id.rawValue,
         envelope.accessToken.accessToken,
-        envelope.gitHubUser.name
+        envelope.gitHubUser.name,
+        envelope.gitHubUser.createdAt.distance(to: now()) < 60*60*24*7 ? 0 : 1
```
Comment (stephencelis) [inline Sources/Database/Database.swift:519]: Limiting to accounts older than a week for now...could tweak this value, though.

**Review by mbrandonw — APPROVED** [review]
very nice, thanks!

File: Sources/Database/Database.swift (hunk 2 above)
Comment (fernandadepaula) [reply Sources/Database/Database.swift]: Jj

**Pull request merged**